rust/cardano-chain-follower/src/chain_sync_live_chains.rs

```diff
@@ -202,11 +203,11 @@ impl ProtectedLiveChainBlockList {
         let last_live_point = Self::get_last_live_point(&live_chain);
         if !previous_point.strict_eq(&last_live_point) {
             // Detected a rollback, so increase the fork count.
-            *fork_count += 1;
+            fork_count.incr();
             let mut rollback_size: u64 = 0;
 
             // We are NOT contiguous, so check if we can become contiguous with a rollback.
-            debug!("Detected non-contiguous block, rolling back. Fork: {fork_count}");
+            debug!("Detected non-contiguous block, rolling back. Fork: {fork_count:?}");
```

Comment:
Maybe `Fork` needs to `impl Display`  and we show names for the Immutable and backfill fork, and something else for live forks, such as:
```
0 = "IMMUTABLE"
1 = "BACKFILL"
2 = "LIVE:{cnt}"
```
so that when we display it, its easier to reason through,  otherwise we are relying on everyone to know 0 = Immutable, 1 = backfile, etc.  The Debug implementation should just show the raw number.  Then use the `Display` here, not `:?`